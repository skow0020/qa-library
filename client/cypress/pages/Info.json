{
    "infoContainer": "#info-container",
    "description": ".carousel-caption"
}