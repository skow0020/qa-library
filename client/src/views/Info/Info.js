import { Container } from "shards-react";
import React from "react";

class Info extends React.Component {
  state = {
    backgroundImage: require("../../images/code1.jpg")
  };
  render() {
    const { backgroundImage } = this.state;

    return (
      <Container>
        <div id="info-container">
          <img className="d-block w-100" src={`${backgroundImage}`} alt="welcome" />
          <div className="carousel-caption d-none d-md-block">
            <h3 className="text-white">This site is a hub of resources for learning programming languages, tools, testing, and best practices across the industry</h3>
            <p>For more information, google it</p>
          </div>
        </div>
      </Container>
    );
  }
}
export default Info;
