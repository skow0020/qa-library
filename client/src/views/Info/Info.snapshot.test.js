import Info from './Info';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Info Snapshot Tests', () => {
  test('Info snapshot', () => {
    const component = renderer.create(
      <Info></Info>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});