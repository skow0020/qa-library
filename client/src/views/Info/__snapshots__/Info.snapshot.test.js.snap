// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info Snapshot Tests Info snapshot 1`] = `
<div
  className="container"
>
  <div
    id="info-container"
  >
    <img
      alt="welcome"
      className="d-block w-100"
      src="code1.jpg"
    />
    <div
      className="carousel-caption d-none d-md-block"
    >
      <h3
        className="text-white"
      >
        This site is a hub of resources for learning programming languages, tools, testing, and best practices across the industry
      </h3>
      <p>
        For more information, google it
      </p>
    </div>
  </div>
</div>
`;
